frontend/src/containers/AddTable.tsx

```diff
@@ -228,28 +285,6 @@ function AddTable() {
     event.stopPropagation();
   };
 
-  const onSelectStaticDataset = async (
-    event: React.FormEvent<HTMLSelectElement>
-  ) => {
-    event.persist();
-    setDatasetLoading(true);
-
-    const jsonFile = (event.target as HTMLInputElement).value;
-    if (jsonFile) {
-      const dataset = await StorageService.downloadJson(jsonFile);
-      setStaticJson(dataset);
-      setCurrentJson(dataset);
-      setStaticDataset(staticDatasets.find((d) => d.s3Key.json === jsonFile));
-    } else {
-      setStaticJson([]);
-      setCurrentJson([]);
-      setStaticDataset(undefined);
-    }
-
-    setDatasetLoading(false);
-    event.stopPropagation();
-  };
-
   const crumbs = [
```

Comment:
You are missing this before the return statement:

`  useEffect(() => {
    if (datasetType) {
      reset({
        datasetType,
      });
    }
  }, []);`

That is important to reset the Static radiobutton when we go to ChooseStaticDataset component and go back to AddTable, check the AddChart component for reference.

Reply:
good catch, will fix